examples/android/src/androidTest/java/org/alohalytics/test/Util.java

```diff
@@ -33,10 +33,16 @@ of this software and associated documentation files (the "Software"), to deal
 
   public static String ReadFileAsUtf8String(String filePath) throws IOException {
     final File file = new File(filePath);
-    final byte[] buffer = new byte[(int) file.length()];
+    final long fileLength = file.length();
+    if (fileLength > Integer.MAX_VALUE) {
```

Comment:
Makes sense. A comment would be nice, perhaps define some `ONE_GIGABYTE` constant?

(I had to double-check with Google what it really is 2GB.)